images/base/files/usr/local/bin/entrypoint

```diff
@@ -203,20 +203,53 @@ fix_cgroup() {
     return
   fi
   echo 'INFO: detected cgroup v1'
-  echo 'INFO: fix cgroup mounts for all subsystems'
+  local current_cgroup
+  current_cgroup=$(grep -E '^[^:]*:([^:]*,)?cpu(,[^,:]*)?:.*' /proc/self/cgroup | cut -d: -f3)
+  if [ "$current_cgroup" = "/" ]; then
+    echo "INFO: cgroupns detected, no need to fix cgroups"
+    return
+  fi
+
+  echo 'WARNING: cgroupns is not enabled! Please run with --cgroupns=private,'
```

Comment:
This might be confusing to end users because the `kind` CLI does not have `--cgroupns` flag.

Reply:
Maybe better to just suggest cgroup v2?

Reply:
As kind is used to test kubernetes, we need to support both v1 and v2, otherwise we won't test it.

What about something like this:
```go
  echo 'WARNING: cgroupns is not enabled! Please use cgroup v2 or cgroup v1 with cgroupns enabled'
```

Reply:
Modified accordingly, PTAL